metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-confluence",
-  "version": "3.0.0-rc0",
+  "version": "3.1.0",		// was 3.0.0-rc1
```

Comment:
Yeah, it wasn't clear to me what the rules are. Is there a doc somewhere I could be referred to? Internally we are testing this with a Puppetfile and I just wanted to increment the version to force r10k to download the newer version. That's not critical though. I'll revert this one.